### PR TITLE
chore(events): Added Event Measuring Service Mesh Performance 101

### LINKED ISSUE #2421: [Event] Measuring Service Mesh Performance 101
Please add the following event to layer5.io/events.

**Event:**
- Event Title: Measuring Service Mesh Performance 101
- Event Link: https://sched.co/siKF
- Talk Description: See the event link
- Speaker: [Navendu Pottekkat](https://layer5.io/community/members/navendu-pottekkat)
---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by Abhijay007**
Signed-off-by: Abhijay Jain <Abhijay007j@gmail.com>

**Description**

Added Event Measuring Service Mesh Performance 101 

This PR fixes #2421 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 826ad914191ee9b751e4ed986a97dc9b5601b81e at: https://61bdce498450846e0f09a6e6--layer5ng.netlify.app

File: src/collections/events/devconf/index.mdx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+thumbnail: ./devconf.png
+eurl: https://sched.co/siKF
+topic: event
+event-category: talk
+published: true
+speakers: ["Navendu Pottekkat"]
+---
+
+<p>
+  This talk answers these questions by empowering attendees with at-hand tooling
```
Comment (leecalcote) [inline src/collections/events/devconf/index.mdx:14]: This description starts without context for what "the questions" are.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit fddf8a556eca0fa174d1ed33a35ecafc03be21d7 at: https://61bf5d3b2671bc319b71091e--layer5ng.netlify.app

File: src/collections/events/devconf/index.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+topic: event
+event-category: talk
+published: true
+speakers: ["Navendu Pottekkat"]
+---
+
+<p>
+Benchmarking a service mesh and your workload’s performance is no simple task.
+
+Questions arise like:
```
Comment (Mr-DG-Wick) [inline src/collections/events/devconf/index.mdx:16]:
```suggestion

```

File: src/collections/events/devconf/index.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+thumbnail: ./devconf.png
+eurl: https://sched.co/siKF
+topic: event
+event-category: talk
+published: true
+speakers: ["Navendu Pottekkat"]
+---
+
+<p>
+Benchmarking a service mesh and your workload’s performance is no simple task.
```
Comment (Mr-DG-Wick) [inline src/collections/events/devconf/index.mdx:14]:
```suggestion
Benchmarking a service mesh and your workload’s performance is no simple task. Questions arise like:
```

File: src/collections/events/devconf/index.mdx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Questions arise like:
+
+Which load generator should you use and what signals should you measure?
```
Comment (Mr-DG-Wick) [inline src/collections/events/devconf/index.mdx:18]: can you please add these questions as an unordered list?

**Review by Mr-DG-Wick — COMMENTED** [review]
hi @Abhijay007 thank you for working on this!
Gave some suggestions that you may look into.

**Comment by Abhijay007** [comment]
@Mr-DG-Wick it's done, is it fine now? 

image

**Comment by l5io** [comment]
🚀 Preview for commit 99837c59548bc02d008d382cc61a3ec80e93be6e at: https://61c04236fafca8d927baf9ce--layer5ng.netlify.app

**Review by Mr-DG-Wick — APPROVED** [review]
LGTM! 🎉 🥳 
Thank you @Abhijay007 !

File: src/collections/events/devconf/index.mdx [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+title: "DEVCONF 2021"
```
Comment (adithyaakrishna) [inline src/collections/events/devconf/index.mdx:2]:
```suggestion
title: "DEVCONF 2022"
```

**Review by adithyaakrishna — CHANGES_REQUESTED** [review]
@Abhijay007 Could you make this small change? Thanks for your contribution 🎉

File: src/collections/events/devconf/index.mdx (hunk 5 above)
Comment (Abhijay007) [reply src/collections/events/devconf/index.mdx:2]: sure !!

**Comment by adithyaakrishna** [comment]
@Abhijay007 could you please check the DCO failing issue?

**Comment by Abhijay007** [comment]
I don't know why these tests are failing? As I already signed DCO

**Comment by adithyaakrishna** [comment]
Can you ping me in Slack?
Let's have a call and I'll help ya with this

**Comment by Abhijay007** [comment]
> @Abhijay007 could you please check the DCO failing issue?

I already signed that but don't know why it is failing

**Comment by Abhijay007** [comment]
> Can you ping me in Slack? Let's have a call and I'll help ya with this

ok sure

**Pull request closed without merging**